src/security.ts

```diff
@@ -54,81 +48,75 @@ export interface AcmCertificateExpirationArgs extends PolicyArgs {
 
 /** @internal */
-export function cmkBackingKeyRotationEnabled(enforcementLevel?: EnforcementLevel): ResourceValidationPolicy {
-    return {
+export const cmkBackingKeyRotationEnabled: ResourceValidationPolicy = {
         name: "cmk-backing-key-rotation-enabled",
         description: "Checks that key rotation is enabled for each customer master key (CMK). Checks that key rotation is enabled for specific key object. Does not apply to CMKs that have imported key material.",
-        enforcementLevel: enforcementLevel || defaultEnforcementLevel,
         validateResource: validateResourceOfType(aws.kms.Key, async (instance, _, reportViolation) => {
             if (!instance.enableKeyRotation) {
                 reportViolation("CMK does not have the key rotation setting enabled");
             }
         }),
     };
-}
+registerPolicy("cmkBackingKeyRotationEnabled", cmkBackingKeyRotationEnabled);
 
 export interface IamAccessKeysRotatedArgs extends PolicyArgs {
     /** Max key age in days. Defaults to 90. */
     maxKeyAge?: number;
 }
 
 /** @internal */
-export function iamAccessKeysRotated(args?: EnforcementLevel | IamAccessKeysRotatedArgs): StackValidationPolicy {
-    const { enforcementLevel, maxKeyAge } = getValueOrDefault(args, {
-        enforcementLevel: defaultEnforcementLevel,
-        maxKeyAge: 90,
-    });
-
-    if (maxKeyAge! < 1 || maxKeyAge! > 2 * 365) {
-        throw new Error("Invalid maxKeyAge.");
-    }
-
-    return {
+export const iamAccessKeysRotated: StackValidationPolicy = {
         name: "access-keys-rotated",
         description: "Checks whether an access key have been rotated within maxKeyAge days.",
-        enforcementLevel: enforcementLevel,
-        validateStack: validateStackResourcesOfType(aws.iam.AccessKey, async (accessKeys, _, reportViolation) => {
+        configSchema: {
+            properties: {
+                maxKeyAge: {
+                    type: "number",
+                    default: 90,
+                },
+            },
+        },
+        validateStack: validateStackResourcesOfType(aws.iam.AccessKey, async (accessKeys, args, reportViolation) => {
+            const { maxKeyAge } =  args.getConfig<IamAccessKeysRotatedArgs>();
             const iam = new AWS.IAM();
             for (const instance of accessKeys) {
                 // Skip any access keys that haven't yet been provisioned or whose status is inactive.
                 if (!instance.id || instance.status !== "Active") {
                     continue;
                 }
-
                 // Use the AWS SDK to list the access keys for the user, which will contain the key's creation date.
                 let paginationToken = undefined;
-
                 let accessKeysResp: AWS.IAM.ListAccessKeysResponse;
                 do {
                     accessKeysResp = await iam.listAccessKeys({ UserName: instance.user, Marker: paginationToken }).promise();
```

Comment:
GH won't let me comment on the line directly, but link 27 has `if (daysSinceCreated > maxKeyAge!) {`. I _think_ the `!` exclamation point can now be removed from `maxKeyAge` after using `Required<T>` above.